apps/sr-frontend/src/app/PageHome.tsx

```diff
@@ -29,6 +48,42 @@ export default function PageHome() {
               >
                 Edit Character
               </Button>
+              <Accordion>
+                <AccordionSummary expandIcon={<ExpandMoreIcon />}>
+                  All target values, if sheet is created
+                </AccordionSummary>
+                <AccordionDetails>
+                  <Stack>
+                    {calc
+                      ?.listFormulas(member0.listing.formulas)
+                      .map((read, index) => {
+                        const computed = calc.compute(read)
+                        const name = read.tag.name || read.tag.q
+                        return (
+                          <Box key={`${name}${index}`}>
+                            <Typography>
+                              {name}: {computed.val}
+                            </Typography>
+                            <Accordion>
+                              <AccordionSummary expandIcon={<ExpandMoreIcon />}>
+                                debug for {name}
+                              </AccordionSummary>
+                              <AccordionDetails>
+                                <Typography component="pre">
+                                  {JSON.stringify(
```

Comment:
Debug code?

Reply:
There's no where else to view this stuff for now. This whole file is temporary anyways